lib/cc/yaml/nodes/mapping.rb

```diff
@@ -75,10 +75,6 @@ def set_warnings(key)
 
       def []=(key, value)
         if mapped_key = mapped_key(key)
-          unless value.is_a? Node
-            node  = subnode_for(mapped_key)
-            value = node if Parser::Ruby.new(value).parse(node)
```

Comment:
I wonder if the intention here is to not replace a node representing `true` with `true` itself: i.e. if the Parser statement succeeds than `value` is the raw form of the existing `node` so let's keep it there...